docs/source/tutorials/himmelblau.ipynb

```diff
@@ -170,6 +170,24 @@
     "agent.plot_objectives()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "75a6a54f-0140-4bd0-8367-0dd14cdf8116",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "{x % 2 for x in np.arange(10)}"
```

Comment:
Is this cell needed? It returns `{0, 1}`.

Reply:
no, I'll fix the notebooks